package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commit-and-tag-version",
-  "version": "12.0.0",
+  "version": "13.0.0",
```

Comment:
I think this version bump might have been included by accident?

Reply:
oops - I must have tested the tool in the wrong repo.